web-ui/src/main/resources/catalog/components/admin/harvester/HarvesterDirective.js

```diff
@@ -130,6 +131,82 @@
     }
   ]);
 
+  /**
+   * Display translation service configuration
+   */
+  module.directive("gnHarvesterTranslateContentConfigurator", [
+    "gnConfig",
+    "gnConfigService",
+    "gnLanguageService",
+    function (gnConfig, gnConfigService, gnLanguageService) {
+      return {
+        restrict: "A",
+        replace: true,
+        scope: {
+          content: "=gnHarvesterTranslateContentConfigurator"
+        },
+        templateUrl: "../../catalog/components/admin/harvester/partials/translate.html",
+        link: function (scope) {
+          scope.xpathExamples = [
+            {
+              xpath:
+                "/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/gmd:CI_Citation/gmd:title",
+              schema: "iso19139"
+            },
+            {
+              xpath:
+                "/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:abstract",
+              schema: "iso19139"
+            },
+            {
+              xpath:
+                "/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:keyword",
+              schema: "iso19139"
+            },
+            {
+              xpath:
+                "/mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:citation/cit:CI_Citation/cit:title",
+              schema: "iso19115-3"
+            },
+            {
+              xpath:
+                "/mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:abstract",
+              schema: "iso19115-3"
+            },
+            {
+              xpath:
+                "/mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:descriptiveKeywords/mri:MD_Keywords/mri:keyword",
+              schema: "iso19115-3"
+            }
+          ];
+          scope.translationProviderConfigured = false;
+          gnConfigService.load().then(function (c) {
+            scope.translationProviderConfigured =
+              gnConfig["system.translation.provider"] !== null &&
+              gnConfig["system.translation.provider"] !== "";
+          });
+
+          scope.languageSource = null;
+          gnLanguageService.getLanguages().then(function (data) {
+            angular.forEach(data, function (lang) {
+              lang.english = lang.label.eng;
+              lang.name = lang.label[scope.lang] || lang.english;
+              lang.tokens = [lang.name, lang.code, lang.english];
+            });
+
+            scope.languageSource = gnLanguageService.getLanguageAutocompleter(data);
+          });
+
+          scope.addXpath = function (xpath) {
+            if (scope.content.translateContentFields.indexOf(xpath.xpath) === -1) {
+              scope.content.translateContentFields += "\n" + xpath.xpath;
+            }
+          };
```

Comment:
Should we unset configuration when unchecking translate mode ? @josegar74 

```suggestion
          };
          
          // Unset configuration when unchecked
          scope.$watch("content.translateContent", function (n, o) {
            if (n != o && n === false) {
              $timeout(function () { // For ui-ace to refresh
                scope.content.translateContentFields = "";
                scope.content.translateContentLangs = "";
              }, 100);
            }
          });
```